docs/en/release_process.md

```diff
@@ -25,36 +37,58 @@ Eg: When creating `v0.1.1` release, will be creating
 the branch `branch-0.1.1`, but for `v0.1.2` we
 would keep using the old `branch-0.1.1`.
 
-In these instructions, I'm referring to an fictitious release `0.1.X`. Change the release version in the examples
-accordingly with the real version.
+In these instructions, I'm referring to an fictitious release `x.y.z`.
+Change the release version in the examples accordingly with the real version.
 
-It is recommended to create a fresh clone of the repository to avoid any local files to interfere in the process:
+It is recommended to create a fresh clone of the repository to 
+avoid any local files to interfere in the process:
 
 ```shell
 $ git clone git@github.com:streamnattive/pulsarctl.git
 $ cd pulsarctl
-$ git checkout -b branch-0.1.X origin/master
-```
 
-#### 2. Create and push the release.
+# Create a branch
+$ git checkout -b branch-x.y.z origin/master
 
-```shell
 # Create a tag
-$ git tag -u $USER@streamnative.io v0.1.X -m 'Release v0.1.X'
-# Update VERSION file
-$ echo v0.1.X > VERSION
+$ git tag -u $USER@streamnative.io vx.y.z -m 'Release vx.y.z'
+```
 
+3. Push the release
+
+```shell
 # Push both the branch and the tag to Github repo
-$ git push origin branch-0.1.X
-$ git push origin v0.1.X
+$ git push origin branch-x.y.z
+$ git push origin vx.y.z
 ```
 
-#### 3. Publish a release note
+4. Publish a release note
 
 Check the milestone in GitHub associated with the release. 
 
-In the released item, add the list of the most important changes that happened in the release and a link to the associated milestone, with the complete list of all the changes. 
+In the released item, add the list of the most important changes 
+that happened in the release and a link to the associated milestone,
+with the complete list of all the changes. 
 
 Update the release draft at [the release homepage of pulsarctl](https://github.com/streamnative/pulsarctl/releases)
 
-Then publish the release draft, the binary and commands doc will auto publish to that release
+Then the release draft, binary, and command doc will be published
+ o that release automatically.
+
+5. Check the release
+
+(1) Visit the Pulsarctl website https://streamnative.io/docs/pulsarctl/vx.y.z/ to
+check the docs have been published successfully. `vx.y.z` is the version of the
+release. For example, https://streamnative.io/docs/publisctl/v0.1.0/.
+
+(2) Use the install command `sh -c "$(curl -fsSL https://raw.githubusercontent.com/streamnative/pulsarctl/master/install.sh)"`
+to install the Pulsarctl and make sure the version of the downloaded Pulsarctl is right.
+
+(3) Add the latest version to the `Available Release` in `README.md`.
+
+(4) Delete the previous milestone.
```

Comment:
why do we delete the previous milestone?

Reply:
I think the old version has been released. There are no pull requests that need to merge to the previous milestone.  So the previous milestone is useless, right?

Reply:
Ah. It should be close, not delete. Will fix